manual/docs/css/custom.css

```diff
@@ -0,0 +1,788 @@
+@import url(http://fonts.googleapis.com/css?family=Cutive+Mono);
```

Comment:
Hm, I'm not sure. It's probably a pretty rare case and it degrades gracefully, so I'd rather keep it simple for now.